webpack.config.js

```diff
@@ -0,0 +1,157 @@
+const CompressionPlugin = require("compression-webpack-plugin")
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const webpack = require('webpack');
+const path = process.cwd();
+const production = process.env.NODE_ENV == 'production';
+
+module.exports = function(opts) {
+  /* Extract Options */
+  opts = opts || {};
+  const entryPoint = opts.entryPoint || '/src/main.js';
+  const outputPath = opts.outputPath || '/assets';
+  const serverBase = opts.serverBase || '/';
+  const publicPath = opts.publicPath || '/assets/';
+  const host = opts.host || '0.0.0.0';
+  const port = opts.port || 8080;
+  const public = opts.public || `${host}:${port}`;
+  const lookupPaths = opts.lookupPaths || [];
+  const parcelifyPaths = opts.parcelifyPaths || [];
+  const index = opts.index;
+
+  /* Parcelify Loader */
+  const parcelifyLoader = {
+    test: /\.js$/,
+    enforce: 'pre',
+    use: ['parcelify-loader'],
+    include: parcelifyPaths.concat([
+      path + '/src'
+    ])
+  };
+
+  /* Babel Loader */
+  const babelLoader = {
+    test: /\.js$/,
+    exclude: /node_modules/,
+    use: {
+      loader: 'babel-loader',
```

Comment:
Have you explored the performance of cacheDirectory?

```
You can also speed up babel-loader by as much as 2x by using the cacheDirectory option. This will cache transformations to the filesystem.
```

Trying to find more information about it

Reply:
tried it but saw no noticeable affect